### PR TITLE
US130301 - implementation of quizzing nav bar

### PR DESCRIPTION
**Pull request by maddiegabriel**
[US130301 ](https://rally1.rallydev.com/#/42960320374d/dashboard?detail=%2Fuserstory%2F605256500003)

Started `_loadQuiz()` implementation to pass the quiz and course name to the navbar.

Needs new constant here: https://github.com/Brightspace/d2l-hypermedia-constants/pull/149

navbarr

### CODE REVIEW

File: components/consistent-evaluation.js [hunk 1]
```diff
@@ -350,11 +350,28 @@ export class ConsistentEvaluation extends LocalizeConsistentEvaluation(LitElemen
 		};
 	}
 
-	async _loadQuiz() {
+	async _loadQuiz(controller) {
 		this._loadingComponents.submissions = false;
 		this._loadingComponents.discussions = false;
```
Comment (maddiegabriel) [inline components/consistent-evaluation.js:355]: will need to add this `_loadingComponents` logic for quizzing

File: components/controllers/ConsistentEvaluationHrefController.js [hunk 2]
```diff
@@ -24,6 +24,35 @@ export class ConsistentEvaluationHrefController {
...
 
+	async getActivityOrganizationName(domainName) {
+		let domainRel;
+
+		switch (domainName) {
+			case 'assignment':
+				domainRel = Rels.assignment;
+				break;
+			case 'quiz':
+				domainRel = Rels.Quizzes.quiz;
```
Comment (maddiegabriel) [inline components/controllers/ConsistentEvaluationHrefController.js:35]: moved this function for the linter - this `'quiz'` case is the only new thing

**Comment by JamieTran** [comment]
Should we be updating [_setTitle](https://github.com/BrightspaceHypermediaComponents/consistent-evaluation/blob/master/components/consistent-evaluation.js#L395) in this PR as well?

**Review by jeroach — APPROVED** [review]
lgtm

**Review by JamieTran — APPROVED** [review]

**Pull request merged**